tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0292.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Unvalidated URL from Deep Link Loaded in WebView
+platform: android
+id: MASTG-TEST-0292
+type: [static]
+weakness: MASWE-0088
```

Comment:
the right MASWE should be [MASWE-0071: WebViews Loading Content from Untrusted Sources
](https://mas.owasp.org/MASWE/MASVS-PLATFORM/MASWE-0071/)

```suggestion
weakness: MASWE-0071
```